superset/connectors/sqla/models.py

```diff
@@ -492,6 +492,7 @@ class SqlaTable(  # pylint: disable=too-many-public-methods,too-many-instance-at
     sql = Column(Text)
     is_sqllab_view = Column(Boolean, default=False)
     template_params = Column(Text)
+    extra = Column(Text)
```

Comment:
We don't need a db migration for this? If so, maybe add a link in this PR to the migration script that has added this column?